react/Header/Header.test.js

```diff
@@ -45,4 +45,8 @@ describe('Header:', () => {
   it('should append returnUrl to signin and register links if present', () => {
     expect(renderHeader({ returnUrl: '/jobs' })).toMatchSnapshot();
   });
+
+  it('should append custom schema', () => {
+    expect(renderHeader({ structuredDataSchema: { org: 'SEEK' } })).toMatchSnapshot();
```

Comment:
Is the `structuredDataSchema` prop meant to be here? It doesn't look like the `Header` component does anything with this, is that right?

Reply:
oops, this needs to be gone!